src/styles/components/nested-service-links.less

```diff
@@ -0,0 +1,26 @@
+/* -----------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+
+NestedServiceLinks
+
+--------------------------------------------------------------------------------
+----------------------------------------------------------------------------- */
+
+.nested-service-link-container {
+  padding-top: @base-spacing-unit * 0.5 !important;
+  padding-bottom: @base-spacing-unit * 0.5 !important;
```

Comment:
I don't have a great solution. But, I do think we should try to override the Canvas specificity by adding more selectors rather than using `!important`. Although it's just another hack to increase the specificity, it's not as bad (IMO) as `!important`